src/app/qgisapp.cpp

```diff
@@ -13061,6 +13062,12 @@ void QgisApp::zoomToBookmarkIndex( const QModelIndex &index )
   mBookMarksDockWidget->zoomToBookmarkIndex( index );
 }
 
+void QgisApp::takeAppScreenShots( const QString &saveDirectory, const int categories )
+{
+  QgsAppScreenShots ass( saveDirectory );
```

Comment:
For maximum LOLs, I think you should utilise the old school "my" prefix here.

Reply:
I've thought about it, but sadly, this goes against our guidelines.